frontend/packages/operator-lifecycle-manager/src/components/index.tsx

```diff
@@ -81,11 +81,13 @@ export const ClusterServiceVersionLogo: React.SFC<ClusterServiceVersionLogoProps
   return (
     <div className="co-clusterserviceversion-logo">
       <div className="co-clusterserviceversion-logo__icon">
-        <img
-          className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
-          src={imgSrc}
-          alt={displayName}
-        />
+        <span className="co-clusterserviceversion-logo__alt_size">
+          <img
+            className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
+            src={imgSrc}
+            alt={displayName}
```

Comment:
Existing problem, but I don't think the `alt` is helpful here since it's repeating the title, which we already show. I would fix this by setting `alt=""` (which is perfectly acceptable for accessibility).

```suggestion
            alt=""
```

Reply:
Well, that is not a fun fix.

Reply:
> Well, that is not a fun fix.

Maybe we should fallback to an animated icon of fireworks instead? :)

Reply:
![fireworks](https://user-images.githubusercontent.com/18728857/81455664-0e7ffe80-914d-11ea-925c-bff4b56a405d.gif)